mlxtend/frequent_patterns/tests/test_apriori.py

```diff
@@ -52,3 +52,13 @@ def test_max_len():
 
     res_df2 = apriori(df, max_len=2)
     assert len(res_df2.iloc[-1, -1]) == 2
+
+
+def test_itemsets_type():
+    res_colindice = apriori(df, use_colnames=False)  # This is defualt behavior
+    for i in res_colindice['itemsets']:
+        assert isinstance(i, set) is True
+
+    res_colnames = apriori(df, use_colnames=True)  # This is defualt behavior
```

Comment:
I think this shouldn't be default behavior then, because the line above with "use_colnames=False" is indicated with default behavior